providers/yttcb/pinniped_addon_secret.lib.yaml

```diff
@@ -0,0 +1,204 @@
+#@ load("@ytt:data", "data")
+#@ load("/lib/helpers.star", "get_no_proxy")
+#@ load("/lib/helpers.star", "get_bom_data_for_tkr_name", "get_image_repo_for_component")
```

Comment:
@shyaamsn per discussion (and related below) this dependency may be able to removed.

Reply:
It was removed.